client/controllers/events/eventIncidentReports.coffee

```diff
@@ -48,6 +49,17 @@ Template.eventIncidentReports.onCreated ->
     <% } %>
   """
 
+  @autorun =>
+    # When user modifies query, set rendering to true which triggers a loading
```

Comment:
I thought about going that route but the incidents view is the only view without a loading indicator that appears when the filtration criteria changes. So I figured we should add one for consistency.